packages/patternfly-4/react-core/src/components/Breadcrumb/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as Breadcrumb } from './Breadcrumb';
```

Comment:
Would change these to:

export * from './Breadcrumb';
export * from './BreadcrumbItem';
export * from './BreadcrumbHeading';

Otherwise you won't be able to access the interfaces in typescript.